docs-2.0/8.service-tuning/load-balance.md

```diff
@@ -0,0 +1,158 @@
++-------------+------+----------+--------------+----------------------+------------------------+
+| Host        | Port | Status   | Leader count | Leader distribution  | Partition distribution |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "storaged0" | 9779 | "ONLINE" | 4            | "nba:4"              | "nba:9"                |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "storaged1" | 9779 | "ONLINE" | 8            | "nba:8"              | "nba:9"                |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "storaged2" | 9779 | "ONLINE" | 3            | "nba:3"              | "nba:9"                |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "storaged3" | 9779 | "ONLINE" | 0            | "No valid partition" | "nba:9"                |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "storaged4" | 9779 | "ONLINE" | 0            | "No valid partition" | "nba:9"                |
++-------------+------+----------+--------------+----------------------+------------------------+
+| "Total"     |      |          | 15           | "nba:15"             | "nba:45"               |
++-------------+------+----------+--------------+----------------------+------------------------+
+Got 6 rows (time spent 849/1420 us)
+```
+
+If any subtask fails, run `BALANCE DATA` again to restart the balancing. If redoing load balancing does not solve the problem, ask for help in the [Nebula Graph community](https://discuss.nebula-graph.io/).
+
+## Stop data balancing
+
+To stop a balance task, run `BALANCE DATA STOP`.
+
+* If no balance task is running, an error is returned.
+
+* If a balance task is running, the task ID is returned.
+
+`BALANCE DATA STOP` does not stop the running subtasks but cancels all follow-up subtasks. The running subtasks continue.
+
+To check the status of the stopped balance task, run `BALANCE DATA <balance_id>`.
+
+Once all the subtasks are finished or stopped, you can run `BALANCE DATA` again to balance the partitions again.
+
+* If any subtask of the preceding balance task failed, Nebula Graph restarts the preceding balance task.
+
+* If no subtask of the preceding balance task failed, Nebula Graph starts a new balance task.
+
+## Remove storage servers
+
+To remove specific storage servers and scale in the Storage Service, use the `BALANCE DATA REMOVE <host_list>` syntax.
+
+For example, to remove the following storage servers:
+
+|Server name|IP|Port|
+|-|-|-|
+|storage3|192.168.0.8|19779|
+|storage4|192.168.0.9|19779|
+
+Run the following statement:
+
+```ngql
+BALANCE DATA REMOVE 192.168.0.8:19779,192.168.0.9:19779;
+```
+
+Nebula Graph will start a balance task, migrate the storage partitions in storage3 and storage4, and then remove them from the cluster.
```

Comment:
A tip:
The raft protocol requires more than half servers alive. You can not remove more than 2 servers.